src/tests/issues.targets

```diff
@@ -3407,4 +3407,164 @@
             <Issue>https://github.com/dotnet/runtime/issues/45568 (workitem PayloadGroup0)</Issue>
         </ExcludeList>
     </ItemGroup>
+
+    <ItemGroup Condition=" $(TargetOS) == 'iOSSimulator' " >
+        <ExcludeList Include = "$(XunitTestBinBase)/Interop/PInvoke/Miscellaneous/MultipleAssembliesWithSamePInvoke/MAWSPITest/**">
+            <Issue>missing assembly</Issue>
```

Comment:
For this and all below, it is recommended to add GH issue links here. If you haven't yet created these, I would suggest updating this file as a subsequent PR once this is merged.

Reply:
That's the plan.

Reply:
It might make sense to create issues prior to adding new iOS/tvOS device lanes to easily build over common failures.